snmp/datadog_checks/snmp/data/default_profiles/nasuni-filer.yaml

```diff
@@ -0,0 +1,334 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-ucd.yaml
+sysobjectid:
+  - 1.3.6.1.4.1.42040.1.1.0
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "nasuni"
+metrics:
+#  - MIB: NASUNI-FILER-MIB
+#    symbol:
+#      OID: 1.3.6.1.4.1.42040.1.5.0
+#      name: filerUpdateAvailable
+# string metric is not supported yet (keep this metric and this comment in profile until it's fixed)
+  - MIB: NASUNI-FILER-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.42040.1.6.0
+      name: filerTotalUnprotectedData
```

Comment:
I think we should add a `nasuni.` prefix for all the metrics since they symbol names here don't have a natural prefix.

Reply:
Makes sense, adding this